.github/actions/spelling/expect.txt

```diff
@@ -19,6 +20,7 @@ apiset
 appinstallertest
 appname
 Archs
+arget
```

Comment:
The spell check was complaining about "arget", but the only reference I could find was "-TargetObject $WinGetSourceListRaw".